book/src/prefabs/how_to_define_prefabs_aggregate.md

```diff
@@ -24,8 +24,7 @@ If you intend to include a [`Component`] that has not yet got a corresponding [`
         Error,
     };
     use serde::{Deserialize, Serialize};
-    use specs_derive::Component;
-    ```
+        ```
```

Comment:
ah sorry, I probably introduced this

Reply:
I'm going to add my approval here because family is in town but pls fix this and push and azriel can merge it for you :)

Reply:
Done